docs/topics/proc_using-openrewrite-recipes.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+
+link:https://docs.openrewrite.org/[OpenRewrite], which is integrated into the {ProductShortName} CLI, uses recipes to automate large-scale, distributed source code refactoring.
+
+You can use OpenRewrite in the {ProductShortName} CLI to prepare Java applications for migration.
+
+[IMPORTANT]
+====
+OpenRewrite recipe support is provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
+====
+
+The `JavaxtoJakarta` recipe renames imported `javax` packages to their `jakarta` equivalents.
+
+.Procedure
+
+. Specify the OpenRewrite recipe when you run `mta-cli`:
++
+[source,terminal]
+----
+$ ./mta-cli --openrewrite "-DactiveRecipes=<recipe name>"  \
+  --input /path/to/source/project --goal dryRun
```

Comment:
I'm not a big fan of "/path/to/source/project". In any case, it should be in <>.